contrib/push_pkg.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+
+set -e
+
+topdir="$(realpath "$(dirname "${0}")/../openwrt")"
+
+# defaults to qemu run script
+ssh_host=localhost
+build_only=0
+preserve_config=1
+
+print_help() {
+	echo "$0 [OPTIONS] PACAKGE_DIR"
+	echo ""
+	echo " -h          print this help"
+	echo " -r HOST     use ssh host (default port will be 22)"
+	echo " -p PORT     use ssh port"
+	echo " -b          build only, do not push anywhere"
+	echo " -P          do not preserve /etc/config. By default, if a package"
+	echo "             defines a config file in /etc/config, this config file"
+	echo "             will be preserved. If you specify this flag, the package"
+	echo "             default will be installed instead."
+	echo ""
+	echo ' To change gluon variables, run e.g. "make config GLUON_MINIFY=0"'
+	echo ' because then the gluon logic will be triggered, and openwrt/.config'
+	echo ' will be regenerated. The variables from openwrt/.config are already'
+	echo ' automatically used for this script.'
+	echo
+}
+
+while getopts "p:r:hbP" opt
+do
+	case $opt in
+		P) preserve_config=0;;
+		p) ssh_port="${OPTARG}";;
+		r) ssh_host="${OPTARG}"; [ -z "$ssh_port" ] && ssh_port=22;;
+		b) build_only=1;;
+		h) print_help; exit 0;;
+		*) ;;
+	esac
+done
+shift $(( OPTIND - 1 ))
+
+[ -z "$ssh_port" ] && ssh_port=2223
+
+if [ "$build_only" -eq 0 ]; then
+	OPENWRT_BOARD="$(ssh -p "${ssh_port}" "root@${ssh_host}" sh -c \''. /etc/os-release; echo "$OPENWRT_BOARD"'\')"
+	OPENWRT_ARCH="$(ssh -p "${ssh_port}" "root@${ssh_host}" sh -c \''. /etc/os-release; echo "$OPENWRT_ARCH"'\')"
```

Comment:
What is the idea behind the OPENWRT_ scope here? I'd say `REMOTE_BOARD` and `REMOTE_ARCH` makes more sense, but I think we'd be fine without the qualifier.